README.md

```diff
@@ -263,9 +263,9 @@ _**Note:** if you're having trouble installing on Mac, try this:_
 
 ##### Checkout branch (optional)
 
-	git checkout <BRANCH_NAME>
+	git checkout -b <BRANCH_NAME>
```

Comment:
Actually `git checkout -b` actually creates a new branch, not checking out an existing branch.  Maybe we could clarify that more (ex. "Checkout existing branch" or "Checkout project branch" instead of just "Checkout branch").  I was working with someone who didn't have any other branches so he had to do `git fetch --all` first, so maybe we should add that before or just add a note that if the branch can't be found you can use that.

Reply:
Since this is found under the **Checkout project** section, I felt the word _existing_ was more appropriate. I also added in the directions regarding what to do if the branch cannot be found.